code/game/objects/structures/crates_lockers/closets/wardrobe.dm

```diff
@@ -128,18 +128,29 @@
 
 /obj/structure/closet/wardrobe/orange
 	name = "prison wardrobe"
```

Comment:
Should this still be called prison wardrobe now that it only contains normal orange jumpsuits?